src/user-icon/user-icon.scss

```diff
@@ -70,6 +70,7 @@
     height: 100%;
     border-width: 0;
     border-radius: 50%;
+    background-color: #FFF;
```

Comment:
While I would have liked to see a dark background on dark theme, I think if someone is using an img it likely is designed for light theme only, so I guess it's ok.
---
Why `#FFF` over `white` tho?

Reply:
we already clear the background in react btw, so most users will never experience this
![image](https://user-images.githubusercontent.com/9084735/143310003-0cdd08e5-57b1-4df8-a585-484f9d451435.png)

Reply:
Updated to use background-1.  This may still introduce issues if the transparent PNG is mostly black, but the same could be said if the transparent PNG was mostly white.  Dev teams can override as needed.

Reply:
This is better, but I wonder if we should just leave it up to the devs to set the background. Currently because of the specificity, it'd be hard to override.

Reply:
It's only `.iui-user-icon > img { background-color: red; }`, right?  My concern is if we don't provide anything, they'll remove the color entirely from the user icon (like they did on the OIDC screen).  If they remove the color & if no picture is provided, we end up with this for initials.
![Screen Shot 2021-12-01 at 11 58 13 AM](https://user-images.githubusercontent.com/849817/144278704-6f21aecf-13fa-4b80-8d31-b2530522cb07.png)![Screen Shot 2021-12-01 at 11 58 07 AM](https://user-images.githubusercontent.com/849817/144278724-224c63fe-a361-454d-8182-1ec48f52aebf.png)